src/project_types.rs

```diff
@@ -1,17 +1,38 @@
 /// Structure to represent a Proton Project.
 /// This is what will be written to a Protonfile at the project root.
-#[derive(Debug, RustcEncodable, RustcDecodable)]
+
```

Comment:
Can you even move the derive? I don't think you can. Though you shouldn't anyway, it would be different from everything else. You can move the doc comment into the struct by changing it to use `//!`, but again, taht would be inconsistent.